lib/core/cardano-wallet-core.cabal

```diff
@@ -232,6 +232,7 @@ test-suite unit
     , network
     , network-uri
     , persistent
+    , regex-pcre-builtin
```

Comment:
I don't know if it builds on windows, but it has the code bundled.

Reply:
`bors try` will tell us this. There's a hydra job that builds the wallet for Windows, from Linux though.